contractcourt/breacharbiter.go

```diff
@@ -47,11 +47,6 @@ var (
 	// procedure, we can recover and continue from the persisted state.
 	retributionBucket = []byte("retribution")
 
-	// justiceTxnBucket holds the finalized justice transactions for all
-	// breached contracts. Entries are added to the justice txn bucket just
-	// before broadcasting the sweep txn.
-	justiceTxnBucket = []byte("justice-txn")
```

Comment:
Adding some additional context here: before anchor commitments, there was basically one way to sweep all outputs in the happy path (ignoring the remote party going to the second level to make things more annoying), so we were able to compute it once, and then just rebroadcast that. Today with anchors, we need to update our breach transaction in the following scenarios:
  * We think we're getting pinned re the HTLC sweeps, so we make another justice transaction that sweeps _just_ the commitment output. 
   * The remote party has broadcasted/confirm a new permutation w.r.t second-level HTLC aggregation (anchors uses sighash single, so they can bundle them as long as they have the same CLTV time lock).
   * One of our possible smaller justice fragment transactions were confirmed, so we'll remove those spend inputs and broadcast another instance

Reply:
I see, that's why we have the variances in the final justice transactions.